mgr/manager.go

```diff
@@ -22,28 +24,39 @@ const DefaultInterval = time.Hour
 // Manager. This defaults to 72 hours.
 const DefaultBefore = time.Hour * 72
 
+// MgrSpecOptions is a compatibility shim for mapping old manager configurables into
+// the common names used by SpecOptions.
+type MgrSpecOptions struct {
+	cert.ParsableSpecOptions
+
+	// OldServiceManagerField is the old yaml configurable name used for this.
+	OldServiceManagerField string `yaml:"service_manager"`
```

Comment:
I'll drop support for these fields in 6+ months, assuming no major complaints from people.  These values should have the same configuration key name as spec's use, thus I'll drop support for these fields sometime after 6 months from now, assuming I'm still active w/ this codebase.